Remove random dash

Jayman2000 opened the pull request:
On the community page it says "Rpm.org does not currently have a specific security process or -personnel." This removes the unnecessary dash.

pmatilai commented:
The dash is not random at all, what it's trying to say is "security process or security personnel" without repeating "security", I don't want anybody mistaking it for saying rpm has no personnel working on it. I'm not a native English speaker so maybe there are better / more legit ways to express that.

Jayman2000 commented:
As a native English speaker, I've never seen a dash used like that. That might still be a valid way of using a dash, but I don't think that the majority of readers will understand what the dash means. I've updated this PR to change the following:

1. I made 'process' plural, and changed 'a' to 'any'. This will make it more obvious that the adjectives at the beginning of the list apply to every item in the list, rather than just the first item. Previously 'a', the first adjective describing the items in the list, could only apply to 'process' as 'personnel' is plural.
2. I put 'personnel' first, and 'processes' second. If the reader misinterprets the sentence, they will read it as saying "rpm does not currently have any specific security personnel" and "rpm does not currently have processes". Saying that rpm does not have processes is vague, doesn't make sense in context, and will hopefully cause the reader to re-read and figure out what the sentence actually is trying to say.

ffesti commented:
Sorry, this should have been merged much earlier. I guess this concatenation of words with dashes really does not work in English.
